src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -156,8 +156,19 @@ public User updateDACUserById(Map<String, User> dac, Integer id) throws IllegalA
             throw new BadRequestException("Institution with the given id does not exist");
         }
 
+        if (Objects.isNull(updatedUser.getStatus())) {
+            updatedUser.setStatus(existingUser.getStatus());
+        }
+
+        Integer statusId = RoleStatus.getValueByStatus(updatedUser.getStatus());
+
+        if (Objects.isNull(updatedUser.getRationale())) {
+            updatedUser.setRationale(existingUser.getRationale());
+        }
+
```

Comment:
By merging the separate update status and rationale functions into a single update user code path, we are allowing the user to update their own status and rationale which is something not previously allowed.